test/spec/contentAvail/contentAvailSrv.spec.js

```diff
@@ -0,0 +1,163 @@
+                    purchaseData.exam['id_' + examIdOrString] = true;
+                }
+            };
+            actions.purchaseDaily = function(dailyOrder){
+                var purchaseData = this.getPurchaseData();
+                purchaseData.daily = dailyOrder;
+            };
+        }])
+    );
+
+    it('when user has subscription then hasSubscription should return true', function () {
+        expect(actions.hasSubscription()).toBeFalsy();
+        actions.addSubscription();
+        expect(actions.hasSubscription()).toBeTruthy();
+    });
+
+    it('when user has subscription then isDailyAvail should return true for all dailies',function(){
+        expect(actions.isDailyAvail(3)).toBeFalsy();
+        actions.addSubscription();
+        expect(actions.isDailyAvail(3)).toBeTruthy();
+    });
+
+    it('given only first daily is free when asking if avail daily should return true only for daily 1',function(){
+        actions.setFreeDaily(1);
+        expect(actions.isDailyAvail(2)).toBeFalsy();
+        expect(actions.isDailyAvail(1)).toBeTruthy();
+    });
+
+    it('given user without purchased exams and without subscription when checking if available section then only return true of it free',function(){
+        actions.setFreeSection(25,1116);
+        expect(actions.isSectionAvail(25,11)).toBeFalsy();
+        expect(actions.isSectionAvail(3,1116)).toBeFalsy();
+        expect(actions.isSectionAvail(25,1116)).toBeTruthy();
+    });
+
+    it('when user has subscription then isSectionAvail should return true for all sections',function(){
+        actions.addSubscription();
+        expect(actions.isSectionAvail(25,11)).toBeTruthy();
+        expect(actions.isSectionAvail(3,1116)).toBeTruthy();
+        expect(actions.isSectionAvail(25,1116)).toBeTruthy();
+    });
+
+    it('given user with purchased exams when checking if available section then return true if exam was purchased',function(){
+        actions.purchaseExam(30);
+        expect(actions.isSectionAvail(30,11)).toBeTruthy();
+        expect(actions.isSectionAvail(30,1116)).toBeTruthy();
+        expect(actions.isSectionAvail(25,1141)).toBeFalsy();
+        expect(actions.isSectionAvail(55,1113)).toBeFalsy();
+    });
+
+    it('given only exam id 25 is free when asking if avail exam should return true only for exam 25',function(){
+        actions.setFreeSection(25,1116);
+        expect(actions.isExamAvail(25)).toBeTruthy();
+        expect(actions.isExamAvail(23)).toBeFalsy();
+    });
+
+    it('given user has subscription when asking isExamAvail then should return true for all exam ids',function(){
+        expect(actions.isExamAvail(3)).toBeFalsy();
+        actions.addSubscription();
+        expect(actions.isExamAvail(3)).toBeTruthy();
+    });
+
+    it('given user purchased daily bundle and free daily is 1 when checking if daily available then only purchased or free dailies should be available',function(){
+        actions.setFreeDaily(1);
```

Comment:
maybe add assestion after this line
expect(actions.isDailyAvail(2)).toBeFalsy();